vaadin-testbench-shared/src/main/java/com/vaadin/testbench/ElementQuery.java

```diff
@@ -316,29 +523,30 @@ public ElementQuery<T> withoutAttribute(String attribute, String value) {
      * <p>
      * For matching the full attribute value, see
      * {@link #withoutAttribute(String, String)}.
+     * For skipping elements containing a substring of the attribute, see
+     * {@link #withoutAttributeContaining(String, String)}.
      *
      * @param attribute
      *            the attribute name
-     * @param value
+     * @param token
      *            the token to look for
      * @return this element query instance for chaining
      *
      * @see #withoutAttribute(String, String)
+     * @see #withoutAttributeContaining(String, String)
      */
-    public ElementQuery<T> withoutAttributeContaining(String attribute, String value) {
-        attributes.add(new AttributeMatch(attribute, "!~=", value));
-        return this;
+    public ElementQuery<T> withoutAttributeContainingToken(String attribute, String token) {
```

Comment:
Example code in Javadoc needs to be fixed

Reply:
Good catch!